### DISCUSSION TITLE
tiered proposal times

### DISCUSSION THREAD

**Discussion by slartibar42** [fetchai/FIPs#6] [Ideas]
I propose amending "FIP-001- Reduce Voting Period to 5 Days" or otherwise implementing a tiered voting time allocation for fetch.ai governance. As proposals can have varied significance on the nature of a project, I suggest that proposals of major importance or gravity receive  a longer voting period than minor issues. This would give the community more time to engage in meaningful discussion, especially the members that may have missed prior deliberations on github, discord, etc. Thank you.

**Comment by rgvnFET** [comment]
Hey @noob2b - thanks for kicking this off. 

When you propose a tiered voting time, did you have any particular time frame in mind?

**Reply by slartibar42** [reply]
Hey @rgvnFET - I think it would be fair enough to have two tiers: one of five days and one of 14. Looking back at the Gnosis - XDAI STAKE merger even the app. 14 day window that was provided hardly gave the community enough  time to digest such a significant decision imo. So I guess there would have to be a list of criteria for the 5 day resp. 14 day category.

**Comment by GataKamsky** [comment]
@noob2b thanks for starting the discussion thread on your suggestion for a tiered governance window. With the addition of steps such as initiating a Github Discussions and starting a FIP process, everyone will have plenty of opportunity to express their views on proposals. This more than mitigates the challenge you highlight. In addition we have also specified the rationale for choosing 5 days in the [FIP-001](https://github.com/fetchai/FIPs/pull/2) body.

A tiered governance approach is also harder to administer when leveraging onchain governance in Cosmos whereas Gnosis (and for that matter Ethereum ecosystem in general) leverages offchain governance.  In case of the Fetch-ai Network, the combination of offchain consensus building using Github discussions and the FIP process plus the onchain governance should be comparable if not better than the Gnosis approach. 

Additionally, if the governance time window of 5 days proves sub-optimal, we always have the option to initiate a new FIP to alter it.

**Reply by slartibar42** [reply]
@GataKamsky thank you for laying out a more detailed view of the governance process on fetch.ai. If Github discussion and FIP reaches the vast majority of the fetch.ai community in time for allowing a meaningful discussion  and can, therefore, ensure a democratic process I support the current proposal and withdraw mine. As a caveat may I, however, add that I cannot find the FIP for the current proposal both in the link to "existing FIPs" (mentioned in the FIP "how to" outline) nor in the FIPs directory. I am not sure if I am just not searching well.

**Reply by GataKamsky** [reply]
We use Github Pull Request feature to publish the FIP in draft mode and conduct the review process. Once the FIP is 'Accepted' i.e. post acceptance in governance proposal for ones needing one which is the case for both [FIP-001](https://github.com/fetchai/FIPs/pull/2) and [FIP-002](https://github.com/fetchai/FIPs/pull/5) you will see these FIPs in the [FIPs folder](https://github.com/fetchai/FIPs/tree/main/FIPs). 

For your reference, I am also attaching screenshots for where you can locate draft FIPs and accepted FIP.

<img width="1153" alt="Screen Shot 2022-07-21 at 10 47 00 PM" src="https://user-images.githubusercontent.com/4752132/180312577-8a601395-4b6d-4f35-b51a-6aa6d461ceb8.png">

<img width="1439" alt="Screen Shot 2022-07-21 at 10 51 14 PM" src="https://user-images.githubusercontent.com/4752132/180314514-b1d5ccd9-e408-42b9-a374-9587d78a8a23.png">

**Reply by slartibar42** [reply]
thank you, that's very helpful. So FIP 001 was created on the 13/7 and voting started the 18th. That would mean app. ten day public deliberation process for proposals in the future? If that is the case, I still feel it to be somewhat short, especially if non devs that work other jobs are to be involved in the discussion and the community significantly grows. Just my personal feeling. Thanks again.

**Reply by GataKamsky** [reply]
No it depends on the FIP. And we can change the duration based on the feedback received during the discussions and the FIP review process. In short, it is flexible.

**Reply by slartibar42** [reply]
Ok, that sounds really good and well designed. Thanks very much for explaining!
